### DISCUSSION TITLE
Setting Json not were it's supposed to be

### DISCUSSION THREAD

**Discussion by thomasTUK** [microsoft/AirSim#3506] [Support Q&A]
Hej there,

I was looking for the setting.json file. As discussed [here ](https://github.com/microsoft/AirSim/discussions/3354#discussioncomment-327739 the default path to the setting file is: /AirSim/settings.json
But it did not find it there. I did find it here:
- \AirSim\docker
- \AirSim\azure\app

Why is that? Can I just copy the settings.json to /AirSim/ ?

Thanks!
Thomas

**Comment by rajat2004** [comment]
Please see https://microsoft.github.io/AirSim/settings/#where-are-settings-stored for info on where the settings.json file is searched for. The first is probably the easiest to work with if dealing with multiple files since you can just specify the exact path. Other way I would recommend is to just modify `Documents/AirSim/settings.json`, there won't be any confusion if you use only this one.
The third option searches the executable folder, however this is generally a very deep location so not used much. https://github.com/microsoft/AirSim/pull/3436 adds another location, the folder from where the executable was launched which is a high-level directory.

 The `AirSim/docker` one is [being copied to `Documents/AirSim`](https://github.com/microsoft/AirSim/blob/master/docker/Dockerfile_binary#L30) when building the docker image, and used when running inside docker. I am currently not aware of how the Azure build works, but possibly it copies the `AirSim/azure/app/settings.json` to the VM as well and then uses it

**Reply by thomasTUK** [reply]
Ah.. I was looking at the wrong folder. I was indeed searching for it at the installation folder but not D:\Users\username\Documents\AirSim

Thanks @rajat2004!
